### DISCUSSION TITLE
missing package in load-path

### DISCUSSION THREAD

**Discussion by eki3z** [emacscollective/borg#113] [Support] [answered]
@tarsius , I enjoy borg's way to manage packages very much. However, when I add a submodule which name ends in .el like
[rg.el](https://github.com/dajva/rg.el), 
```sh
git submodule add --name rg.el https://github.com/dajva/rg.el lib/rg.el
```

borg cann't add it to `load-path` automatically.

I usually choose to remove .el suffix, so I don't encounter this problem until I forget once. So does it a bug or just design in this way? Maybe this deserve a notice in borg or packed's mannual.

**Comment by tarsius** [comment, marked as answer]
The `.` ends up being used in the name of Git variables where it serves as a separator. `x.y.el.z` is very different from `x.y-el.z`. I'm not sure it is even valid. Doesn't seem worth adding a workaround for this.

**Reply by eki3z** [reply]
I see. A note like "omit .el suffix when add submodule" in mannual would be better.

**Reply by tarsius** [reply]
There are just too many details that one could mention. If we mentioned them all, then the manual would become unreadable because of all the little helpful hints.
